style.css

```diff
@@ -1,30 +1,55 @@
 
 * {box-sizing: border-box;}
 
-.footer, .header, .main {
+.header,
+.main, 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Mandri_Firsttask/pullRequest?prid=5482251)